### DISCUSSION TITLE
How to Instal Plugins?

### DISCUSSION THREAD

**Discussion by Johnnyblu84** [Rem0o/FanControl.Releases#2059] [Q&A]
How can I install plugins? I have searched everywhere and all I can find is put files in the plugins folder, right-click, properties & check to unblock the files. I did all that and Fancontrol will not detect anything. Trying this with the Thermaltake plugin

Please any help would be much appreciated.

**Comment by Rem0o** [comment]
Are you running the 4.8 version or 7.0 of net framework?

**Reply by Johnnyblu84** [reply]
I tried with both, nothing is showing up in plugins. At first I didn't have any net framwork installed, then I saw your reply. So I tried 4.8 restarted my pc nothing changed, I uninstalled 4.8 and installed 7.0 and tried again and still nothing. Strange
